ts/input/tex/FindTeX.ts

```diff
@@ -180,6 +180,8 @@ export class FindTeX<N, T, D> extends AbstractFindMath<N, T, D> {
                 braces++;
             } else if (match[0] === '}' && braces) {
                 braces--;
+            } else if (match[0].substr(0) === '%') {
```

Comment:
`substr` is deprecated
```suggestion
            } else if (match[0].substring(0) === '%') {
```